### ISSUE TITLE
v4.8.0 breaking alternative route json schema in node-osrm

### ISSUE THREAD

**Issue by freenerd** [Project-OSRM/osrm-backend#1693]
(I looked at the node-osrm code but couldn't see a change that may impact output format, thus I'm opening the ticket here.)

I'm running the following node-osrm code on [an extract of Berlin](http://download.geofabrik.de/europe/germany/berlin.html):

```js
var OSRM = require('osrm');
var osrm = new OSRM();

query = { 
    zoomLevel: 18,
    printInstructions: true,
    coordinates:  [
      [ '52.5483512878418', '13.355409622192383' ],
      [ '52.54133685682971', '13.34787368774414' ] 
    ],
   alternateRoute: true 
};

osrm.route(query, function(err, result) {
    console.log(result);
}
```

In node-osrm v4.7.1 this produces the following output ([full gist](https://gist.github.com/freenerd/651dd83c4fa7eb790338)):

```json
{
    "alternative_instructions": [
        [
            [
                "10", "Ostender Straße", 8, 0, 17, "8m", "NE", 37, 1
            ],
            "..."
        ]
    ]
}
```

In node-osrm v4.8.0 this produces the following output ([full gist](https://gist.github.com/freenerd/ea54a3f6e36eefc3aa0f))

```json
{
    "alternative_instructions": [
        [
                "10", "Ostender Straße", 8, 0, 17, "8m", "NE", 37, 1
        ],
        "..."
        []
    ],
}
```

The important differences:
  - v4.8.0 has one less level of array depth (which kind-of makes sense, since we only allow for one alternative anyways, right)
  - v4.8.0 adds an empty last instruction at the end (which is certainly a bug)

If the array format change is on purpose, that this is a breaking change that should be communicated in the [node-osrm changelog](https://github.com/Project-OSRM/node-osrm/blob/master/CHANGELOG.md). Also this should have certainly been caught by unit tests :crying_cat_face: 

/cc @TheMarex @daniel-j-h

**Comment by freenerd** [comment]
https://github.com/Project-OSRM/osrm-backend/commit/a6b44a14709bf760425cfef288d1f14a6914f569 fixed this for me. I guess we are looking at a quick 4.8.1 release here ...

**Comment by TheMarex** [comment]
This is fixed in `4.8.1`.
